saleor/dashboard/order/views.py

```diff
@@ -374,6 +375,7 @@ def remove_order_voucher(request, order_pk):
                             ctx, status=status)
 
 
+@shared_task
```

Comment:
I'm not sure what happened here.

1. This is a view. I'm not sure how views would work as Celery tasks.
2. `shared_task` is meant for reusable packages.
3. You never call `task.delay(...)` so making these functions tasks is a no-op.

Reply:
Ok, so we should have separate method with just pdf creation part and this method should have @celery.task decorathor?

Reply:
Celery is asynchronous. If you delegate a task to Celery you can't assume it will be done before you send the response.